packages/wallet/backend/src/rafiki/service.ts

```diff
@@ -500,18 +499,7 @@ export class RafikiService implements IRafikiService {
     return receivedQuote
   }
 
-  public getRates(base: string): RatesResponse {
-    const exchangeRates: Rates = {
-      USD: {
-        EUR: 0.908936
-      },
-      EUR: {
-        USD: 1.10019
-      }
-    }
-    return {
-      base,
-      rates: exchangeRates[base] ?? {}
-    }
+  public async getRates(base: string): Promise<RatesResponse> {
```

Comment:
Do we need this function in rafiki.service? maybe the caller can directly call from ratesService.

Reply:
indeed.